### ISSUE TITLE
Not expected value in debug watch

### ISSUE THREAD

**Issue by skilliscode** [OmniSharp/omnisharp-vscode#1093]
## Environment data
.NET Command Line Tools (1.0.0-preview2-1-003177)

Product Information:
 Version:            1.0.0-preview2-1-003177
 Commit SHA-1 hash:  a2df9c2576

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
VS Code version:
C# Extension version:
1.6.2
## Steps to reproduce
run `dotnet new` and put the following code into the Program.cs file
```csharp
using System;

namespace ConsoleApplication
{
    public class Program
    {
        public static void Main(string[] args)
        {
            var format = System.Globalization.CultureInfo.InvariantCulture.DateTimeFormat;
            var d1 = DateTime.Parse("1/1/2017 20:00", format);
            var d2 = DateTime.Parse("1/1/2017 21:30", format);
            Console.WriteLine(d1<d2);
        }
    }
}
```
Set a break point at the `Console.WriteLine(d1<d2);` and run debug, add `d1<d2` in watch. 

## Expected  behavior
d1<d2 returns true

## Actual behavior
d1<d2 returned false

**Comment by rajkumar42** [comment]
@skilliscode Thanks for reporting the issue!

**Comment by gregg-miskelly** [comment]
This is a dupe of https://github.com/OmniSharp/omnisharp-vscode/issues/753. The issue is fixed in the 1.2 version of CoreCLR.
